src/main/kotlin/no/nav/tilleggsstonader/sak/opplysninger/oppgave/OppgaveService.kt

```diff
@@ -29,6 +48,168 @@ class OppgaveService(
         )
     }
 
+    fun opprettOppgave(
+        behandlingId: UUID,
+        oppgavetype: Oppgavetype,
+        tilordnetNavIdent: String? = null,
+        beskrivelse: String? = null,
+        mappeId: Long? = null,
+        prioritet: OppgavePrioritet = OppgavePrioritet.NORM,
+    ): Long {
+        val oppgaveFinnesFraFør = getOppgaveFinnesFraFør(oppgavetype, behandlingId)
+        return if (oppgaveFinnesFraFør !== null) {
+            oppgaveFinnesFraFør.gsakOppgaveId
+        } else {
```

Comment:
Jeg synes egentlige at dette er veldig rar logikk, selv om jeg vet at det er kopi. 
Si eks at det skal opprettes 2 oppgaver av en gitt type på behandling 👀 Denne tabell er ju ikke nødvendigvis oppdatert med riktig status for "ferdigstilt"

Ønsker kanskje en oppfølgningsoppgave på dette.. ?